src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -196,6 +244,146 @@ class TFLongformerQuestionAnsweringModelOutput(ModelOutput):
     global_attentions: Optional[Tuple[tf.Tensor]] = None
 
 
+@dataclass
+class TFLongformerSequenceClassifierOutput(ModelOutput):
```

Comment:
Those new outputs need to be added in `longformer.rst` under the section "Longformer specific outputs" (there are plenty of examples there to help :-) )